lisa/runner.py

```diff
@@ -33,6 +33,19 @@ def parse_testcase_filters(raw_filters: List[Any]) -> List[schema.BaseTestCaseFi
     return filters
 
 
+def print_results(
+    test_results: List[Union[TestResult, TestResultMessage]],
+    output_method: Callable[[str], None]
+    ) -> None:
+    for index, result in enumerate(test_results):
+        # Printing for TestResultMessage type.
+        if hasattr(result, "name"):
+            output_method(f"[{index}] {result.status.name:<8} {result.message}")
+        # Printing for TestResult type
+        else:
+            output_method(f"[{index}] {result.id_} {result.message}")
```

Comment:
@squirrelsc I wasn't sure how to how to print for the TestResult type so this is what I put for now. Please let me know how you want the output to look for TestResult